src/main/java/seedu/address/model/item/Name.java

```diff
@@ -56,4 +56,8 @@ public int hashCode() {
         return fullName.hashCode();
     }
 
+    @Override
+    public int compareTo(Name o) {
+        return fullName.compareTo(o.fullName);
+    }
```

Comment:
Not an issue, just that this is the first time I have seen o as a variable name 😆